beauty-diff.sh

```diff
@@ -14,42 +14,46 @@ js-beautify -v >/dev/null 2>&1 || { echo >&2 "Please install js-beautify: https:
 colordiff -v >/dev/null 2>&1 || { echo >&2 "Please install colordiff: https://www.colordiff.org/"; exit 1; }
 ###################################
 
+if [[ $# -eq 0 ]]; then
+  echo "usage: $(basename "$0") file"
+  exit 1
+fi
+
 # Get input filename
-FILE_PATH=("${@}")
+FILE_PATH="${*}"
```

Comment:
I'm trying to go with the spirit of what you had before, but as the script stands now, my first pick is actually
```suggestion
FILE_PATH="$1"
```
I wasn't sure if you were going for the ability to specify a `file with spaces` on the command line as `beauty-diff file with spaces` rather than `beauty-diff "file with spaces"`.  See #2 for thoughts on handling multiple files.